bulk/management/commands/data_quality.py

```diff
@@ -0,0 +1,343 @@
+        total_votes_without_voters = bills.filter(from_organization=chamber, votes__votes=None).values_list("votes").count()
+        # votes (which do have voters, as to not include above category)
+        #   but where yes/no count do not match actual voters
+        total_votes_where_votes_dont_match_voters = 0
+        bills_with_votes_with_voters = bills.filter(from_organization=chamber).exclude(votes__votes=None)
+        for b in bills_with_votes_with_voters:
+            for vote_object in b.votes.all():
+                total_yes = 0
+                total_no = 0
+                total_absent = 0
+                total_excused = 0
+                total_not_voting = 0
+                total_other = 0
+
+                voter_count_yes = 0
+                voter_count_no = 0
+                voter_count_absent = 0
+                voter_count_excused = 0
+                voter_not_voting = 0
+                voter_count_other = 0
+
+                vote_counts = vote_object.counts.all()
+                votes = vote_object.votes.all()
+                # Parsing through vote_counts
+                for count in vote_counts:
+                    if "yes" == count.option:
+                        total_yes = count.value
+                    elif "no" == count.option:
+                        total_no = count.value
+                    elif "absent" == count.option:
+                        total_absent = count.value
+                    elif "excused" == count.option:
+                        total_excused = count.value
+                    elif "not voting" == count.option:
+                        total_not_voting = count.value
+                    elif "other" == count.option:
+                        total_other = count.value
+                    else:
+                        print("Other option found in vote_counts: ", count)
+                        print(count.__dict__)
+                # Parsing through voters and adding up their votes
+                for voter in votes:
+                    if voter.option == "yes":
+                        voter_count_yes += 1
+                    elif voter.option == "no":
+                        voter_count_no += 1
+                    elif voter.option == "absent":
+                        voter_count_absent += 1
+                    elif voter.option == "excused":
+                        voter_count_excused += 1
+                    elif voter.option == "abstain":
+                        voter_not_voting += 1
+                    elif voter.option == "other":
+                        voter_count_other += 1
+                    else:
+                        print(voter.option)
+                # Checking to see if votes and vote counts match
+                if (voter_count_yes != total_yes) or (voter_count_no != total_no) or (voter_count_absent != total_absent) or (voter_count_excused != total_excused) or (total_not_voting != voter_not_voting) or (total_other != voter_count_other):
```

Comment:
let's simplify this to something like

(voter_count_yes != 0 and voter_count_yes != total_yes) or ...

Reply:
Updated.